### ISSUE TITLE
Release 0.9.9

### ISSUE THREAD

**Issue by gberche-orange** [mevansam/terraform-provider-cf#113]
The team agreed to have consensus on when to cut such release through a github issue.

Now that the provider rename #44 is complete, I'd like to cut a 0.9.9 release so that users such as Orange can start providing feedback before the 1.0 is cut. 

This requires a release so that automation built on top of TF can pick the released provider (e.g. in https://github.com/orange-cloudfoundry/cf-ops-automation/issues/80)

**Comment by jcarrothers-sap** [comment]
I agree that this makes sense.  I feel we should agree to rename the repository to `terraform-provider-cloudfoundry` as part of making this release in order to reflect the name change and ensure that the binary is built with the correct name going forward.

**Comment by janosbinder** [comment]
I have to add renaming does not have any implication. We tried this internally on a project, and _github_ redirects `git` operations to the renamed repo, also on the website the `terraform-provider-cf` will be redirected automatically to `terraform-provider-cloudfoundry`.

**Comment by jcarrothers-sap** [comment]
@gberche-orange I feel we should move forward with this release next week after the next batch of merges.
For example, I believe #143 is an issue that has already been fixed on our dev branch.

**Comment by pedrodparkes** [comment]
Can't reproduce on DEV branch. Can be closed.
Good luck with the release!
